netdata-installer.sh

```diff
@@ -1211,7 +1211,7 @@ govercomp() {
   for ((i = 0; i < ${#ver1[@]}; i++)); do
     if [ "${ver1[i]}" -gt "${ver2[i]}" ]; then
       return 1
-    elif [ "${ver1[i]}" -gt "${ver2[i]}" ]; then
+    elif [ "${ver2[i]}" -gt "${ver1[i]}" ]; then
```

Comment:
I _could_ be wrong here but I"m not sure you can do this kind of comparison when it comes to versions as you end up effectively doing lexicographical comparisons and this may be problematic. It would be better if you parsed the version strings into numbers like I've done elsewhere in other parts of hte netdata installer and updater and compared actual integers. 🤔

Reply:
it works

https://github.com/netdata/netdata/blob/bdda59ce541dc0be0092328ba74f01f057871989/netdata-installer.sh#L1198-L1220

the function expectes go.d.plugin version

https://github.com/netdata/netdata/blob/bdda59ce541dc0be0092328ba74f01f057871989/netdata-installer.sh#L1186-L1195

Reply:
> you parsed the version strings into numbers

i do this - regex + split (IFS=.). We could use some other functions you mentioned, i am ok with it.